File: esercizi_java/test/compito_python/veicolo.py
class veicolo:
    def __init__(self, cod, targa, modello, marca, cost):
        self.cod = cod
        self.targa = targa
        self.modello = modello
        self.marca = marca
        self.cost = cost

    def toString(self):
        return self.targa + "\t" + str(self.cod) + "\t" + self.modello + "\t" + self.marca + "\t" + str(self.cost) + "\t"

File: esercizi_java/NetBeansProjects/ciaone/src/gestione/veicolo.java

package gestione;

/**
 *
 * @author tobiasacchetto
 */
public abstract class veicolo {
    protected int cod;
    protected String targa, modello, marca;
    protected double cost;
    
    
    public veicolo(int cod, String targa, String modello, String marca, double cost){
    this.cod = cod;
    this.targa = targa;
    this.modello = modello;
    this.marca = marca;
    this.cost = cost;
    }
    
    
    @Override
    public String toString(){
    return targa + "\t" + cod + "\t"+ modello + "\t" + marca+"\t" + cost+ "\t";
    }
    
}
